doc/classes/RenderingDevice.xml

```diff
@@ -92,6 +104,7 @@
 			<param index="0" name="compute_list" type="int" />
 			<param index="1" name="compute_pipeline" type="RID" />
 			<description>
+				Tells the GPU what compute pipeline to use when processing the compute list. If the shader has changed since the last time this function was called, Godot will unbind all descriptor sets and will be re-bind them inside [method compute_list_dispatch].
```

Comment:
```suggestion
				Tells the GPU what compute pipeline to use when processing the compute list. If the shader has changed since the last time this function was called, Godot will unbind all descriptor sets and will re-bind them inside [method compute_list_dispatch].
```